libcontainer/factory_linux.go

```diff
@@ -137,10 +136,6 @@ type LinuxFactory struct {
 	// a container.
 	InitArgs []string
 
-	// CriuPath is the path to the criu binary used for checkpoint and restore of
-	// containers.
-	CriuPath string
-
```

Comment:
I feel like it's still kinda make sense to keep it in case user want to package criu together with runc and named it `runc-criu` for example. But im not sure who even does it that way :D